fix(xo-server): import OVA with broken VMDK size in metadata

fbeauchamp opened the pull request:
ova generated from oracle virtualization server seems to have the size of the vmdk
instead of the disk size in the metadata
this will cause the transfer to fail when the import try to write data
after the size of the vmdk, for example a 50GB disk make a 10GB vmdk. It will fail
when import reach data in the 10-50GB range

### Description

_Short explanation of this PR (feel free to re-use commit message)_

### Checklist

- Commit
  - Title follows [commit conventions](https://bit.ly/commit-conventions)
  - Reference the relevant issue (`Fixes #007`, `See xoa-support#42`, `See https://...`)
  - If bug fix, add `Introduced by`
- Changelog
  - If visible by XOA users, add changelog entry
  - Update "Packages to release" in `CHANGELOG.unreleased.md`
- PR
  - If UI changes, add screenshots
  - If not finished or not tested, open as _Draft_

julien-f commented on CHANGELOG.unreleased.md:
```diff
@@ -29,6 +29,7 @@
 
 <!--packages-start-->
 
+- xo-vmdk-to-vhd patch
```
Rebase on top of `master` and make sure `xo-server` is listed.

fbeauchamp committed:
fix(xo-server): import ova with broken vmdk size in metadata

ova generated from oracle virtualization server seems to have the size of the vmdk
instead of the disk size in the metadata
this will cause the transfer to fail when the import try to write data
after the size of the vmdk, for example a 50GB disk make a 10GB vmdk. It will fail
when import reach data in the 10-50GB range

julien-f committed:
Update CHANGELOG.unreleased.md

julien-f committed:
Update CHANGELOG.unreleased.md
